frontend/src/components/dashboard/create/govsg/RadioChoice.tsx

```diff
@@ -26,7 +26,10 @@ function RadioChoice({
         <label htmlFor={id}>{label}</label>
       </div>
       {rest.checked && children && (
```

Comment:
If `children = []`, will there be an error here? Since `[]` is truthy but we can't do `children[0]`

Reply:
Thank you for the catch, Kishen.

I'll add a length check in a subsequent PR. Hope that's ok!

Reply:
Added to #2162